config.yaml.default

```diff
@@ -13,10 +13,12 @@
 # limitations under the License.
 
 ---
-auth:
-  github-token: your-github-token
-  circleci-token: your-circleci-token
-# circleci token is needed only if you are going to use any
-# circleci-related functionality
-organization: googleapis
-repo-name-regex: 'nodejs-.*'
+github-token: your-github-token
+circleci-token: your-circleci-token
+repos:
+  - googleapis/nodejs-.*
+  - GoogleCloudPlatform/^cloud-[a-z]*-nodejs$
```

Comment:
@alexander-fenster @theacodes I am a little torn on what config to support here.  While this style of just listing out the repos with regex is short and clean, it *does* have 2 drawbacks:
1. There's no way to express different CircleCI keys for each org (see below)
2. The regex-ed-ness here is hella sloppy.  The regex only applies to the repo name, the part *after* the first `/`.  That could trip folks up.  

Alternatively, we could offer a syntax like this:

```yaml
github-token: ....
orgs:
  - googleapis
     repoNameRegex: nodejs-.*
     circleciToken: ....
  - google
     repos:
       - google-api-nodejs-client
       - google-auth-library-nodejs
       ....
```

I went with the shorter syntax because for most people that will ever use this thing, it's fine.  If we want to be ultra expressive, the longer format is more technically correct. 

Looking for thoughts or strong opinions :)

Reply:
The fact that regex applies only to the repo name, but not the org name, is very weird. How about this:

```yaml
repos:
  - org: googleapis
    repos: nodejs-.*
  - org: google
    repos: google-auth-library-nodejs
  - org: google
    repos: google-api-nodejs-client
```

which is equivalent to your shorter syntax but without the regex application drawback (and without extra split by `/` in the code).

Reply:
That's an improvement!  Do you think there are any problems with assuming the `repos` field is a regex?

Reply:
So basically we should point that each section makes a loop over all organization repositories, since GitHub API cannot filter repositories by regex and we must do it on our side by iterating. Is it possible to have two options:

```yaml
repos:
  - org: googleapis
    regex: nodejs-.*    # iterates and filters
  - org: google
    name: google-auth-library-nodejs   # no iteration, just get the required repo
```
If we go with regex-by-default as it is now, requesting `google/google-auth-library-nodejs` will first get a list of all `google` repos which is huge.